app/uk/gov/hmrc/gform/wshttp/WSHttpModule.scala

```diff
@@ -16,14 +16,23 @@
 
 package uk.gov.hmrc.gform.wshttp
 
+import akka.actor.ActorSystem
+import com.typesafe.config.Config
+import play.api.{ Configuration, Play }
 import play.api.http.HttpVerbs.{ POST => POST_VERB }
 import uk.gov.hmrc.gform.auditing.AuditingModule
 import uk.gov.hmrc.gform.config.ConfigModule
 import uk.gov.hmrc.gform.playcomponents.PlayComponents
 
-class WSHttpModule(auditingModule: AuditingModule, configModule: ConfigModule, playComponents: PlayComponents) {
+class WSHttpModule(
+  auditingModule: AuditingModule,
+  configModule: ConfigModule,
+  playComponents: PlayComponents,
+  theActorSystem: ActorSystem) {
 
   val auditableWSHttp: WSHttp = new WSHttp {
+    override protected def actorSystem: ActorSystem = theActorSystem
```

Comment:
Why was this change needed? It doesn't seems related with removing existing functionality?